packages/openjdk-17/build.sh

```diff
@@ -0,0 +1,118 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/PojavLauncherTeam/mobile
+TERMUX_PKG_DESCRIPTION="Java development kit and runtime"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=17.0
+TERMUX_PKG_REVISION=9
+TERMUX_PKG_SRCURL=https://github.com/termux/openjdk-mobile-termux/archive/2a42d94548738280653d48e55ee8891fdd87b656.tar.gz
+TERMUX_PKG_SHA256=3c86157f56ffa837c66f46a778736627be2c149f3fc55588005c88cc488518bd
```

Comment:
https://github.com/termux/openjdk-mobile-termux/pull/1 is merged. Change commit and sha-256 here to pick the new sources.